dashboard/src/scenes/organisation/DragAndDropSettings.js

```diff
@@ -0,0 +1,260 @@
+import React, { useState, useEffect, useRef, useCallback, useMemo } from 'react';
+import SortableJS from 'sortablejs';
+import { toast } from 'react-toastify';
+import ButtonCustom from '../../components/ButtonCustom';
+import { ModalContainer, ModalBody, ModalFooter, ModalHeader } from '../../components/tailwind/Modal';
+import { capture } from '../../services/sentry';
+
+/*
+data: [
+  {
+    groupTitle: string,
+    items: [any]
+  }
+]
+*/
+
+const DragAndDropSettings = ({
```

Comment:
Autant utiliser la vraie notation JSDoc plutôt que la notre. Elle a l'avantage de s'afficher dans VSCode. Et c'est un standard. C'est habituellement ce qui est utilisé pour typer du JS sans Typescript (et ça permet une migration douce mais ça c'est une autre histoire).

<img width="560" alt="Capture d’écran 2023-01-30 à 10 41 36" src="https://user-images.githubusercontent.com/1575946/215441930-30848016-53bd-448b-b5a9-45a162416b49.png">

J'aurai pu mettre toutes les propriétés, mais je veux pas faire de forcing.

```suggestion
/**
 * @typedef {Object} DragAndDropSettingsProps
 * @property {Array<{groupTitle: string, items: Array}>} data
 */
/**
 * @param {DragAndDropSettingsProps} props
 * @returns {JSX.Element}
 */
const DragAndDropSettings = ({
```